FaynaTown.Tennis.Planner.Bot.Core/Providers/JsonExtensionProvider.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.IO;
+
+namespace FaynaTown.Tennis.Planner.Bot.Core.Providers;
+internal class JsonExtensionProvider : IFileExtensionProvider {
+    public string ChangeExtension(string path) => Path.ChangeExtension(path, ".json");
```

Comment:
Little format refactoring suggestion: if we use new line approach for lambda methods everywhere, we use it everywhere :)

```suggestion
    public string ChangeExtension(string path) =>
        Path.ChangeExtension(path, ".json");
```

Reply:
Done